CMakeLists.txt

```diff
@@ -4,6 +4,9 @@ project(libiio C)
 set(LIBIIO_VERSION_MAJOR 0)
 set(LIBIIO_VERSION_MINOR 12)
 set(VERSION "${LIBIIO_VERSION_MAJOR}.${LIBIIO_VERSION_MINOR}")
+if (WIN32)
+	string(TIMESTAMP BUILD_YEAR "%Y")
+endif()
```

Comment:
curios: do we need this for non-Win32 builds ? or just for Win32 ?

Reply:
The WIN32 variable is defined on Windows, Cygwin and MinGW, independently of the CPU architecture (stupid, I know).